src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+
+    protected String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+        System.out.print("Got it. I've added this task:\n" + this + '\n');
```

Comment:
Any reason why you have not included ".by" with 'this' keyword? I think it would be better practice to specify which attribute the calling object is accessing and not to just directly access this object.

Reply:
I actually want it to print the .toString value every time. Will be adding .toString behind them.